protocol-dashboard/src/hooks/useDashboardWalletUsers.ts

```diff
@@ -17,12 +17,17 @@ const dashboardWalletUsersBatcher = create({
   scheduler: windowScheduler(10)
 })
 
+export const getDashboardWalletUserQueryKey = (wallet: string) => {
+  return ['dashboardWalletUsers', wallet]
+}
+
 export const useDashboardWalletUser = (wallet: string) => {
   return useQuery({
```

Comment:
we do have redux on the dashboard, but I didn't think using Audius-query made sense here because we don't have/need a normalized cache for this. using react-query keeps things abstracted away and simple